iaso/api/payments/views.py

```diff
@@ -570,14 +570,18 @@ class PaymentsViewSet(ModelViewSet):
 
     """
 
-    http_method_names = ["patch", "get"]
+    http_method_names = ["patch", "get", "options", "post"]
```

Comment:
Yup, I wasn't aware of that, but [this is DRF default behavior](https://github.com/encode/django-rest-framework/blob/8e304e1adbb0f99f91a15ed3abd379104bba3b89/rest_framework/metadata.py#L78)… and I don't know why.

So we have two solutions:

1. do what you've done
1. create a subclass of `SimpleMetadata` and override `determine_actions()`

The only problem with your solution is that it could trick the auto-doc tools like swagger etc.

Reply:
I think if we plan to use that pattern widely in the future, the second solution looks more robust